decentraland/unity-renderer
# Fix/bug video throttled even if close

## What does this PR change?

Fix video throttling as described in issue #3247

There were a couple of bugs that prevented the correct notification of scene change.
One was the overflow of subscriptions to events regarding parcel change and scene change.
The second and most important fix was about an oversight on the SceneController which was not checking the scene position properly.

## How to test the changes?

To test the changes, you need to check if the videos are being throttled when you enter the scene containing them, using this link:
[https://play.decentraland.zone/?renderer-branch=fix%2Fbug-video-throttled-even-if-close&NETWORK=mainnet&position=53%2C53&CATALYST=peer-testing.decentraland.org&realm=testing&DEBUG_SCENE_LOG=](url)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
